packages/webapp/src/containers/Help/saga.js

```diff
@@ -31,12 +32,26 @@ export function* supportFileUploadSaga({ payload: { file, form } }) {
     const formData = new FormData();
     formData.append('_file_', file);
     formData.append('data', JSON.stringify(form));
-    const result = yield call(axios.post, supportUrl(), formData, {
-      headers: {
-        'Content-Type': 'multipart/form-data',
-        Authorization: 'Bearer ' + localStorage.getItem('id_token'),
-      },
-    });
+    let result;
+    if (form.support_type === 'Delete my farm') {
```

Comment:
I'm not a big fan of this being a string that needs to coincidentally match with the value defined in the form, but I didn't see an already established pattern for handling this situation